src/MouseTooltip.jsx

```diff
@@ -28,10 +35,18 @@ class MouseTooltip extends React.PureComponent {
   }
 
   getTooltipPosition = ({ clientX: xPosition, clientY: yPosition }) => {
+    const screenWidth = window.innerWidth;
+    const screenHeight = window.innerHeight;
+    const containerWidth = this.containerRef.current.clientWidth;
+    const containerHeight = this.containerRef.current.clientHeight;
```

Comment:
These too, please? :wink:

```suggestion
    const { clientWidth: containerWidth, clientHeight: containerHeight } = this.containerRef.current;
```